packages/react-instantsearch-hooks-web/README.md

```diff
@@ -1,3 +1,49 @@
 # react-instantsearch-hooks-web
 
 > 🚧 This version is not yet production-ready.
+
+React InstantSearch Hooks is an open-source, **experimental UI library** for React that lets you quickly build a search interface in your front-end application.
```

Comment:
the experimental fits here, until we have the docs, right? or would you prefer it seeing removed and changed to beta before release?

Reply:
If we're shipping the doc before the release announcement, I don't mind the warning being removed now.

Reply:
```suggestion
React InstantSearch Hooks is an open-source, UI library for React that lets you quickly build a search interface in your front-end application.
```

Reply:
I believe we're good removing the experimental version because we've already removed the warning inside the library.